modules/v1/workspaces/controllers/WorkspaceController.php

```diff
@@ -98,4 +99,37 @@ public function actionInviteUsers()
         $dbTransaction->commit();
         return $this->response(null, 201);
     }
+
+    /**
+     * Get users by workspace
+     *
+     * @return array|mixed
+     * @throws \Exception
+     */
+    public function actionGetUsers()
+    {
+        $workspaceId = Yii::$app->request->get('id') ?? false;
```

Comment:
@ll-bat You don't need to use this statement `Yii::$app->request->get('id') ?? false`. Get function in request function by default returns null, therefore you can just have: `$workspaceId = Yii::$app->request->get('id')`